book/website/communication/peer-review/peer-review-overview.md

```diff
@@ -0,0 +1,26 @@
+(cm-pr-overview)=
+
+# Peer Review Types
+
+
+## Levels of Peer Review
+* *Step 1* - Peer review often happens informally among team members.
+Feedback helps improve your project, on its way to being shared with the broader community.​
+* *Step 2* - The project is going well, but it needs input from the broader community to reach it's full potential.
+This stage is often where the formal peer review process happens, but in an open project this level of feedback is often provided throughout.​
+* *Step 3* - By incorporating everyone's ideas, the project unites the community and percieves feedback as a way to improve projects rather than as a competitive process.
+
+We see peer review as an important exercise in [collaboration](We need to add links to collaboration here) not limited to a quality-safeguard procedure.
+However, in current research practice, peer review often appears as a hurdle everyone needs to pass through.
+
+## Types of Peer Review
+​
+Peer review is a critical component of advancing research. Types of peer review include:
+* manuscripts
+* software (including code, documentation, and examples)
+* communication and presentation materials
+
+
+For manuscripts, one of the added values of a journal articles, compared to, for instance, a preprint is that it has been [peer reviewed and modified by the author in response to the reviewer's suggestions](here we need to ling to guidlines). 
```

Comment:
I would maybe move this sentence up to the section you had earlier "Peer review can be formal or informal and ideally occurs at different levels throughout a project. In the traditional publishing model, there was a lot of focus on the peer review process of a journal article."
The Open access chapter also defines preprints if you want to link to that

Reply:
I am not sure I fully understood, but I did something )))